AutomationFramework/src/test/java/steam/steps/GamePageSteps.java

```diff
@@ -0,0 +1,32 @@
+package steam.steps;
+
+import framework.pageElements.Label;
+import framework.pageElements.ListOfElements;
+import org.openqa.selenium.By;
+
+import java.util.ArrayList;
+import java.util.logging.Logger;
+
+public class GamePageSteps {
+    public static final Logger logger = Logger.getLogger(MainPageSteps.class.getName());
+    private final Label nameOfGame = new Label(By.xpath("//div[@class=\"apphub_AppName\"]"));
+    private final ListOfElements listOfTagsOfGame = new ListOfElements(By.xpath("//div[@class=\"glance_tags popular_tags\"]/a"));
+    private final Label reviewsStatusOfGame = new Label(By.xpath("//span[contains(@class,'game_review_summary positive')]"));
+    private final Label priceOfGame = new Label(By.xpath("//div[@class='discount_final_price']"));
```

Comment:
Why you have locators in steps? Move them to pages. Steps should interact with elements via pages not bare locators